src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -2,68 +2,64 @@
 @model APIViewWeb.Pages.Assemblies.ReviewModel
 @{
     ViewData["Title"] = "Review";
+    string user = User.FindFirst(c => c.Type == "urn:github:login")?.Value;
```

Comment:
No likey, add an extension method (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods) to ClaimsPrincipal to get Github login so you can write `User.GetGithubLogin()`